bgpd/bgp_fsm.c

```diff
@@ -1512,11 +1565,62 @@ static int bgp_connect_success(struct peer *peer)
 			zlog_debug("%s passive open", peer->host);
 	}
 
+	/* Send an open message */
 	bgp_open_send(peer);
 
 	return 0;
 }
 
+/* TCP connection open with RFC 4271 optional session attribute DelayOpen flag
+ * set. */
+static int bgp_connect_success_w_delayopen(struct peer *peer)
+{
+	if (peer->fd < 0) {
+		flog_err(
+			EC_BGP_CONNECT,
+			"bgp_connect_success_w_delayopen peer's fd is negative value %d",
```

Comment:
can we change bgp_connect_success_w_delayopen to `%s/__func__` combination?

Reply:
yes.